samples/.NET/NUnit/PlaywrightTestWithArtifact.cs

```diff
@@ -34,8 +34,12 @@ public virtual BrowserNewContextOptions ContextOptions()
         [SetUp]
         public async Task Setup()
         {
-            // Create Browser, context and page
-            Browser = await CreateBrowser(BrowserType);
+            // Connect Remote Browser using BrowserType.ConnectAsync
+            var playwrightService = new PlaywrightService();
```

Comment:
how about putting an if condition based on a flag from runsettings file if flag is true then remote connect 

if false then business as usual normal way of connecting